_posts/2025-08-15-this-month-in-servo.md

```diff
@@ -352,6 +352,20 @@ categories:
       windows; fix building on windows 11
 -->
 
+Servo has smashed its record again in July, with **367 pull requests** landing in our nightly builds!
+This includes several new web platform features:
+
+- **APNG** and **WebP animated images**, in addition to GIF (@rayguo17, #37637)
+- **&lt;source width>** and **&lt;source height>** (@leo030303, @xiaochengh, #38152)
+- **‘image-rendering’ property** now affects **‘border-image’** (@lumiscosity, @Loirooriol, #38346)
+- **TextDecoderStream** (@minghuaw, #38112)
+- **replace() method** on **CSSStyleSheet** (@stevennovaryo, #38244)
+- **texImage3D() method** on **WebGL2RenderingContext** (@wusyong, #37718)
+- **convertToBlob()** and **transferToImageBitmap() methods** on **OffscreenCanvas** (@tharkum, #37786, #37880)
+- **adoptedStyleSheets property** on **Document** and **ShadowRoot**, with `--pref dom_adoptedstylesheet_enabled` (@stevennovaryo, #38163)
+- **sessionStorage** is now isolated from localStorage (@janvarga, #37803)
```

Comment:
`- **sessionStorage** is now isolated per webview, and cloned as part of executing the `open() method` on `Window` (@janvarga, #37803)`

Reply:
done to that effect in 5bb00dd8712a9fdc56b7820742e7c65e9f439c6f